src/main/java/com/jadajabajava/services/EmployerService.java

```diff
@@ -2,25 +2,29 @@
 
 import com.jadajabajava.entities.Employer;
 import com.jadajabajava.repositories.EmployerRepository;
+import lombok.NonNull;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.MessageSource;
 import org.springframework.stereotype.Service;
 
 import java.util.Optional;
 
 @Slf4j
 @Service
-public class EmployerService extends AbstractService<Employer> {
+public class EmployerService {
 
-    private final EmployerRepository employerRepository;
+    private final EmployerRepository repository;
+
```

Comment:
Codacy found an issue: [Perhaps 'messageSource' could be replaced by a local variable.](https://app.codacy.com/gh/jada-jaba-java/job-seeker-service/pullRequest?prid=5720132)